Components/Dark Mode/Button/style.scss

```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css?family=Work+Sans:400,700,900');
+
+$anim-text-offset: 1em;
+
```

Comment:
Remove unwanted files. Compile this code to CSS and delete the rest of the files which are not in use.